compiler/luci-interpreter/src/CMakeLists.txt

```diff
@@ -57,5 +57,5 @@ endif(NOT ENABLE_TEST)
 
 nnas_find_package(GTest REQUIRED)
 
-GTest_AddTest(buddy_manager_test ${TEST_SOURCES})
-target_link_libraries(buddy_manager_test ${LUCI_INTERPRETER_BINARY})
+GTest_AddTest(luci_interpreter_memory_manager_test ${TEST_SOURCES})
+target_link_libraries(luci_interpreter_memory_manager_test ${LUCI_INTERPRETER_BINARY})
```

Comment:
unintended replace? maybe add new gtest...

Reply:
I intentionally renamed the test. I'd like to test memory managers in the single gtest, rather than make separate gtest for all memory managers. This will reduce duplicate codes, and I think it is more aligned with other tests (there are `luci_interpreter_kernels_test`, `luci_interpreter_loader_test`).